### PR TITLE
Add page and page_size to find query

### PR DESCRIPTION
**Pull request by mandruis7**
This is in order to go over the 20 objects that We've with the default limit in both page_size (limit 100) and page for iterate in case that you've more than 100 objects

### CODE REVIEW

File: supersetapiclient/base.py [hunk 1]
```diff
@@ -258,7 +258,7 @@ def get(self, id: int):
 
         return object
 
-    def find(self, **kwargs):
+    def find(self, page_size=100, page=0, **kwargs):
```
Comment (opus-42) [inline supersetapiclient/base.py]: Can you add type hints here please ?

File: supersetapiclient/base.py [hunk 2]
```diff
@@ -271,13 +271,18 @@ def find(self, **kwargs):
...
             }
             params = {
                 "q": json.dumps(query)
             }
         else:
-            params = {}
+            query = {"page_size": page_size, "page": page}
+            params = {
+                "q": json.dumps(query)
+            }
```
Comment (opus-42) [inline supersetapiclient/base.py]: I think you should factorise that and initialise the query before the if... else (line 266)

```python
       query = {
                "page_size": page_size,
                "page": page
       }

        if kwargs != {}:
        
        ...
        
             query = query.update({...})
             params = ...
```

**Comment by mandruis7** [comment]
I've updated the MR with some updates with this new code the if ... else It's no longer required and I'm as well initializing the query as you requested

**Review by opus-42 — APPROVED** [review]

**Comment by opus-42** [comment]
Thank you @nogreintel 😄

**Pull request merged**